text/0000-selecting-default-java-version.md

```diff
@@ -0,0 +1,34 @@
+# Selecting the next default Java Version for Paketo Buildpacks
+
+## Summary
+
+This RFC introduces a process to select the next default version of the Paketo Java Buildpack.
+
+## Motivation
+
+Currently, there is no process how the default version of Java is picked in the Paketo world. As of today (January 2023), the default still is Java 11 (released September 2018) although Java 17 (released September 2021) is already available.
+
+## Detailed Explanation
+
+The default should be changed according to a defined process rather than at some undefined point in time to some undefined version. To do so, the default should be changed once the latest releases LTS version of Java is at least one year old. 
+
+## Rationale and Alternatives
+
+- Do nothing, update default manually from time to time
+- Always use the latest LTS immediately
+  - Could break people immediately
+- This RFC
+  - Gives room to adapt to breaking changes
+
+## Implementation
+
+To fulfill this RFC, the default version should be changed to Java 17. All future versions should be bumped according to this RFC.
+
+## Prior Art
+
+n/a
+
+## Unresolved Questions and Bikeshedding
+
+- Define the time to wait after a new LTS was published
+- Should this be an automated process?
```

Comment:
My vote would be no. I don't see this as being a ton of work, so I'm not sure if it meets the payoff for automating. That said, if someone wants to automate it and submit a PR with a Github Action to do that, then we would accept it.

Even if it's not a full automation, like if it just tracks upstream and opens an issue when a year's time has elapsed, that would be helpful too.